spec/stack.js

```diff
@@ -19,4 +18,59 @@ describe('Stack', () => {
         .to.alter(() => myStack.length(), { from: 0, to: 1 })
     })
   })
+
+  context('pop()', () => {
+    it('returns and removes element from top of the stack.', () => {
+      const myStack = new Stack()
+      myStack.push('foo')
+      myStack.push('foo')
+
+      expect(() => myStack.pop())
+        .to.alter(() => myStack.length(), { from: 2, to: 1 })
+    })
+
+    it('returns null if stack is empty.', () => {
+      const myStack = new Stack()
+
+      expect(myStack.pop()).to.equal(null)
+    })
+  })
+
+  context('peek()', () => {
+    it('returns element at top of the stack.', () => {
```

Comment:
A test should be included that tests for the behavior of `peek` when the `Stack` is empty.